public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -199,12 +218,14 @@ export class CloudWatchDatasource extends DataSourceApi<CloudWatchQuery, CloudWa
             throw { message: 'Invalid extended statistics' };
           }
 
+          const reducedItem = _.pick(item, metricsParams);
```

Comment:
I have a small concern how we can be sure that the above list is complete therefore I would prefer to filter out the internal to explorer variables instead if this is feasible.